src/OpenLoco/src/GameCommands/GameCommands.h

```diff
@@ -719,6 +778,47 @@ namespace OpenLoco::GameCommands
         doCommand(GameCommand::changeCompanyColourScheme, regs);
     }
 
+    struct ChangeCompanyColourSchemeArgs
+    {
+        ChangeCompanyColourSchemeArgs() = default;
+        explicit ChangeCompanyColourSchemeArgs(const registers& regs)
+            : companyId(CompanyId(regs.dl))
+            , isPrimary(regs.ah)
```

Comment:
this should move to the body of the constructor and do `if (setColourMode == 0) { isPrimary = regs.ah == 0; }`

Reply:
only the isPrimary initialization, right?

Reply:
and if setColourMode is not 0?

Reply:
ah, i use the value from al, right?